bridge/test/sharedConstants/4-validators-snapshots-100-Group1.ts

```diff
@@ -0,0 +1,1010 @@
+import { BigNumber } from "ethers";
+import { ValidatorRawData } from "../ethdkg/setup";
+import { SignedBClaims, Snapshot } from "../setup";
+
+export const validatorsSnapshotsG1: ValidatorRawData[] = [
```

Comment:
Really dumb question: Is it possible to generate this data at test runtime instead of having to check in? With all the magic values here it's entirely inscrutable to anyone without deep understanding.

Reply:
Unfortunately no. To generate this data we need a lot of the ETHDKG cryptographic and threshold signature functions that only exist in golang at the moment.